README.md

```diff
@@ -1,3 +1,7 @@
+# Use the [lerna template](https://github.com/salesforcecli/lerna-template) if you need a library and plugin
```

Comment:
Is it just me that finds the 3 titles distracting?

What if we did as the first main header, `Using this Plugin Template` which would contain this and everything up to `Everything past here is only a suggestion as to what should be in your specific plugin's description`. The message would be the last thing before `# plugin-&lt;REPLACE ME&gt;` which would be the start of the actual README that goes in the plugin?

Thoughts?

Reply:
I updated them to be smaller font, but still in the same place - I think it looks fine this way 🤷‍♂️